core/controllers/suggestion.py

```diff
@@ -688,16 +1131,19 @@ def _upload_suggestion_images(files, suggestion, filenames):
     # TODO(#10513): Find a way to save the images before the suggestion is
     # created.
     for filename in filenames:
-        image = files.get(filename)
-        image = base64.decodebytes(image.encode('utf-8'))
+        image = files[filename]
+        decoded_image = (
+            image if isinstance(image, bytes)
+            else base64.decodebytes(image.encode('utf-8'))
```

Comment:
Why is there now ambiguity here if there wasn't before? Can we standardize it to either str or bytes depending on what is currently being accepted (when you try it using the local dev server)?

Reply:
> Can we standardize it to either str or bytes depending on what is currently being accepted (when you try it using the local dev server)?

Yes, we can use str here because from the frontend the string value is coming, Also a proof is attached for the same.

https://user-images.githubusercontent.com/12553576/201693941-20a6fcc4-b13c-4a7d-81b0-6c8993c44603.mp4